data/frame.go

```diff
@@ -175,6 +175,7 @@ func NewFrame(name string, fields ...*Field) *Frame {
 	return &Frame{
 		Name:   name,
 		Fields: fields,
+		RefID:  "",
```

Comment:
Why do why need this? RefID should already by zero-initialized to an empty string?

Reply:
Realized I missed reading the last part of your comment. While I understand the reason, setting a zero-initialized value explicitly would rather make me think twice whether there's a reason to.

Reply:
It's just explicit. That's all. We don't need it.

Reply:
I can remove this one.